src/components/Search/SearchResultItem.tsx

```diff
@@ -1,21 +1,64 @@
-import React from 'react'
+import React, { useCallback, useState } from 'react'
 import { ImageSize, IResultItem } from '../../types/SearchTypes';
 import SearchResultImage from './SearchResultImage';
+import Card from '@mui/material/Card';
+import Grid from '@mui/material/Grid'
+import Typography from '@mui/material/Typography'
+import CardMedia from '@mui/material/CardMedia'
+import CardContent from '@mui/material/CardContent'
+import { CardActions } from '@mui/material';
+import Button from '@mui/material/Button';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import { application } from 'express';
 
 interface ISearchResultItemProps {
   item: IResultItem,
-  imageSize?: ImageSize
+  imageSize?: ImageSize,
+  deviceId: string
 }
 
 function SearchResultItem(props: ISearchResultItemProps) {
-  const { item } = props
+  const { item, deviceId } = props
   const imageSize = props.imageSize ?? 'md'
+  const payload = JSON.stringify({
+    "context_uri": item.uri,
+    "offset": {
+      "position": 0
+    },
+    "position_ms": 0
+  });
+
+  const handleSubmit = useCallback((event) => {
+    const getResponse = async () => {
+      try {
+        const response = await fetch(`/api/v1/me/player/play?device_id=${deviceId}`,
+                                    { method:'PUT',
+                                      body:payload, 
+                                      headers:{
+                                        'Content-Type': 'application/json'
+                                      }
+                                    });
+      } finally {
+        // Reset search text after searching
```

Comment:
We can omit the finally block altogether

In the Search component it was helpful to always reset our search text whether or not we got an error or not.

Long term it would be a good idea to catch an error here and display some sort of message to let the user know something went wrong but that may be out of scope of this change

Reply:
This has been reviewed, I changed the comment in the finally block to hint at errors in the future. Decided to keep the finally block as a reminder and as precedent. Thanks again.